ConfigurationSystem/Client/CSAPI.py

```diff
@@ -411,12 +411,12 @@ def addShifter( self, shifters = None ):
     :return: S_OK/S_ERROR
     """
 
-    def getOpsSection( ):
+    def getOpsSection():
```

Comment:
Here using S_ERROR or exception is pretty much equivalent and will take the same amount of code. Absolutely no advantage of using exception. But we should be consistently using conventions of the project. Uniform style does matter in a multi-developer project. We discussed this zillions of times and let's not restart it.